GliaWidgets/SecureConversations/TranscriptModel.DividedChatItemsForUnreadCount.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+extension SecureConversations.TranscriptModel {
+    static func dividedChatItemsForUnreadCount(
```

Comment:
would it make sense to simplify the function name to something "addDividerToChatItems"?

Reply:
I moved this part to address linter warning. I would prefer to leave it as it is for now, to avoid adding more changes to this PR including renaming in unit tests (the whole test case).